script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_2.txt

```diff
@@ -61,21 +61,21 @@ Arcueid going through that suffering.
 All of the blame lies on me, the one who killed her―――
 <Page17>
 "Shiki. Don't make the same mistake twice.
-The extinguishing of weaker creatures is the natural way of things.
+The extinguishing of weaker creatures is the natural way of things.//to be rechecked
```

Comment:
Don't see anything wrong with this sentence.

I'd personally rearrange it to 'It's only natural that weaker creatures are extinguished.'

Reply:
Extinguishing was the problem here iirc. Disappearance was suggested but is still wrong apparently

Reply:
The two recommendations were "extinction" and "disappearance". I'm like generally OK with extinction because it at least fits with in the evolutionary model, but IMO this doesn't need a change.